tests/cli/cloud/test_cloud.py

```diff
@@ -1121,481 +1121,3 @@ def test_set_workspace_with_less_than_10_workspaces(respx_mock):
         PREFECT_API_URL: bar_workspace.api_url(),
         PREFECT_API_KEY: "fake-key",
     }
-
-
-def test_cannot_get_webhook_if_you_are_not_logged_in():
```

Comment:
Only changes to this file are that I moved the webhook tests out into a separate test_webhook module that more closely mirrors the code-under-test directory structure. Think the webhook CLI was at one point nested all under one file as well, was broken out later, but then these tests stayed as they are.

I added another cloud submodule so thought i'd break this out while i was here.